data.sql

```diff
@@ -1,5 +1,32 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', date '2020-02-03', 0, '1', 10.23);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', date '2018-11-15', 2, '1', 8);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', date '2021-01-7', 1, '0', 15.04);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', date '2017-05-12', 5, '1', 11);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', date '2020-02-08', 0, '0', -11);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', date '2021-12-15', 2, '1', -5.7);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', date '1993-04-02', 3, '0', -12.13);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', date '2005-06-12', 1, '1', -45);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', date '2005-06-07', 7, '1', 20.4);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', date '1998-10-13', 3, '1', 17);
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', date '2022-05-14', 4, '1', 22);
+
+
+INSERT INTO owners (full_name, age) VALUES ('Sam smith', 34);
+INSERT INTO owners ( full_name, age) VALUES ('Jennifer Orwell', 19); 
+INSERT INTO owners ( full_name, age) VALUES ('Bob', 45); 
+INSERT INTO owners ( full_name, age) VALUES ('Melody Pond', 77); 
+INSERT INTO owners ( full_name, age) VALUES ('Dean Winchester', 14); 
+INSERT INTO owners ( full_name, age) VALUES ('Jodie Whittaker', 38); 
+INSERT INTO species ( name) VALUES ('Pokemon');
+INSERT INTO species ( name) VALUES ('Digimon');
```

Comment:
###  Kindly make sure you add the following data 👇🏻 
- [ ] Insert the following data for vets:
 - Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
 - Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
 - Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
 - Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
- [ ] Insert the following data for specialties:
 - Vet William Tatcher is specialized in Pokemon.
 - Vet Stephanie Mendez is specialized in Digimon and Pokemon.
 - Vet Jack Harkness is specialized in Digimon.

- [ ] Insert the following data for visits:
 - Agumon visited William Tatcher on May 24th, 2020.
 - Agumon visited Stephanie Mendez on Jul 22th, 2020.
 - Gabumon visited Jack Harkness on Feb 2nd, 2021.
 - Pikachu visited Maisy Smith on Jan 5th, 2020.
 - Pikachu visited Maisy Smith on Mar 8th, 2020.
 - Pikachu visited Maisy Smith on May 14th, 2020.
 - Devimon visited Stephanie Mendez on May 4th, 2021.
 - Charmander visited Jack Harkness on Feb 24th, 2021.
 - Plantmon visited Maisy Smith on Dec 21st, 2019.
 - Plantmon visited William Tatcher on Aug 10th, 2020.
 - Plantmon visited Maisy Smith on Apr 7th, 2021.
 - Squirtle visited Stephanie Mendez on Sep 29th, 2019.
 - Angemon visited Jack Harkness on Oct 3rd, 2020.
 - Angemon visited Jack Harkness on Nov 4th, 2020.
 - Boarmon visited Maisy Smith on Jan 24th, 2019.
 - Boarmon visited Maisy Smith on May 15th, 2019.
 - Boarmon visited Maisy Smith on Feb 27th, 2020.
 - Boarmon visited Maisy Smith on Aug 3rd, 2020.
 - Blossom visited Stephanie Mendez on May 24th, 2020.
 - Blossom visited William Tatcher on Jan 11th, 2021.